package/kedro_viz/models/graph.py

```diff
@@ -636,7 +652,7 @@ def load_latest_metrics_data(
             return json.load(fs_file)
 
     @staticmethod
-    def load_metrics_versioned_data(
+    def load_tracking_versioned_data(

```

Comment:
I know this follows the old naming convention, but I think it reads better as `load_versioned_tracking_data`. WDYT?